### PR TITLE
Add missing API v2 filters

### PR DESCRIPTION
**Pull request by johngian**
* Language
* Account
* Group
* Skill

### CODE REVIEW

**Comment by johngian** [comment]
@akatsoulas can you take a look at this?

**Comment by akatsoulas** [comment]
@johngian sure!

File: mozillians/users/tests/test_api_v2.py [hunk 1]
```diff
@@ -196,3 +197,22 @@ def test_filter_emails_alternate(self):
         f = UserProfileFilter(request.GET, queryset=UserProfile.objects.all())
         eq_(f.qs.count(), 1)
         eq_(f.qs[0], user.userprofile)
```
Comment (akatsoulas) [inline mozillians/users/tests/test_api_v2.py]: Shouldn't you add a few more tests for the rest of the filters?

Comment (johngian) [inline mozillians/users/tests/test_api_v2.py]: We are not doing something else custom except of this so I suppose we don't need to check ``django-filters`` functionality nor ``django lookups``.

**Comment by akatsoulas** [comment]
Looking good r+

**Pull request merged**